modules/machineset-non-guaranteed-instance.adoc

```diff
@@ -3,30 +3,21 @@
 // * machine_management/creating_machinesets/creating-machineset-aws.adoc
 
 [id="machineset-non-guaranteed-instance_{context}"]
-= MachineSet Spot Instances
+= MachineSets that deploy machines as Spot Instances
 
-You can save on costs by creating an AWS MachineSet that deploys machines as non-guaranteed Spot Instances.
-Spot Instances utilize unused AWS EC2 capacity and are less expensive than On-Demand Instances.
-You can use Spot Instances for workloads that can tolerate interruptions, such as batch or stateless,
-horizontally scalable workloads.
+You can save on costs by creating a MachineSet running on AWS that deploys machines as non-guaranteed Spot Instances. Spot Instances utilize unused AWS EC2 capacity and are less expensive than On-Demand Instances. You can use Spot Instances for workloads that can tolerate interruptions, such as batch or stateless, horizontally scalable workloads.
```

Comment:
s/utilize/use/

Reply:
> s/utilize/use/

@bobfuru Good suggestion to use "use" instead of "utilize, but do you think this sounds okay to say "Spot Instances use unused AWS EC2 capacity"?

Reply:
Ah didn't catch that, good question! In that case, I might have said, "Spot Instances use available AWS EC2 capacity".